content/00.front-matter.md

```diff
@@ -4,9 +4,16 @@
 
 {% for author in authors %}
 + **{{author.full_name}}**<br>
-  [\@{{author.github_username}}](https://github.com/{{author.github_username}}) ·
-  [{{author.orcid}}](https://orcid.org/{{author.orcid}}) ·
-  {{author.affiliations}} ·
-  Funded by {{author.funders}}
+  {% if author.github_username != 'None' -%}
+    [\@{{author.github_username}}](https://github.com/{{author.github_username}}) ·
```

Comment:
Should the ` · ` be prepended to included items. Right now you can get trailing interpuncts?